Editor/Resources/EditorWindow/Components/InfoLinks.uxml

```diff
@@ -0,0 +1,15 @@
+﻿<!-- Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. -->
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+
+<ui:UXML xmlns:ui="UnityEngine.UIElements">
+    <ui:Style src="./InfoLinks.uss"/>
+    <ui:VisualElement class="separator separator--horizontal separator--small">
+        <ui:Label name="InfoLinkDocumentationLink" class=""/>
```

Comment:
Can we remove the blank class declarations?

Reply:
removed